core/src/main/scala/akka/MethodBang_template.scala

```diff
@@ -3,17 +3,17 @@ package akka
 import java.io.File
 
 import akka.actor._
-import akka_debugging.DistributedStackTrace
-import akka_debugging.collector.Collector.{RelationMessage, CollectorMessage}
-import akka_debugging.collector.DatabaseCollector
 import com.typesafe.config.ConfigFactory
-import org.aspectj.lang.{JoinPoint, ProceedingJoinPoint}
+import org.aspectj.lang.ProceedingJoinPoint
 import org.aspectj.lang.annotation._
+import pl.edu.agh.iet.akka_debugging.DistributedStackTrace
+import pl.edu.agh.iet.akka_debugging.collector.Collector.{CollectorMessage, RelationMessage}
+import pl.edu.agh.iet.akka_debugging.collector.DatabaseCollector
 
 import scala.util.Random
 
 
-@Aspect
+//@Aspect
```

Comment:
The template could be moved to resources and then the ugly unpacking can be avoided though we will be implementing changes to this in another task.